src/workerd/jsg/setup.h

```diff
@@ -310,8 +310,12 @@ class Isolate: public IsolateBase {
     // constructing a `Lock` on the stack.
 
   public:
-    Lock(const Isolate& isolate, V8StackScope& scope)
+    Lock(const Isolate& isolate, V8StackScope&)
         : jsg::Lock(isolate.ptr), jsgIsolate(const_cast<Isolate&>(isolate)) {
+      // `V8StackScope` must be provided to prove that one has been created on the stack before
+      // taking a lock. Any GC'dp ointers stored on the stack must be kept within this scope in
```

Comment:
```suggestion
      // taking a lock. Any GC'd pointers stored on the stack must be kept within this scope in
```

Reply:
Thanks but if I commit this I'll have to wait through CI again... typos just aren't worth fixing a lot of the time. :/